tests/test_ffimachine.py

```diff
@@ -21,14 +21,14 @@ def test_ffi_from_file():
     assert isinstance(ffi, Machine)
     # test attributes have the right shapes
     assert ffi.time.shape == (1,)
-    assert ffi.flux.shape == (1, 33812)
+    assert ffi.flux.shape == (1, 33840)
     assert ffi.flux_2d.shape == (1, 180, 188)
-    assert ffi.flux_err.shape == (1, 33812)
-    assert ffi.column.shape == (33812,)
-    assert ffi.row.shape == (33812,)
-    assert ffi.ra.shape == (33812,)
-    assert ffi.dec.shape == (33812,)
-    assert ffi.sources.shape == (259, 13)
+    assert ffi.flux_err.shape == (1, 33840)
+    assert ffi.column.shape == (33840,)
+    assert ffi.row.shape == (33840,)
+    assert ffi.ra.shape == (33840,)
+    assert ffi.dec.shape == (33840,)
+    assert ffi.sources.shape == (259, 15)
```

Comment:
this changed because saturated pixels are treated now. Before they were removed before initializing `machine`, now they still are in the `self.flux` array but they are masked out in the source masks.